pkg/proxy/net/packetio_test.go

```diff
@@ -268,17 +269,25 @@ func TestKeepAlive(t *testing.T) {
 	stls, ctls, err := security.CreateTLSConfigForTest()
 	require.NoError(t, err)
 	frontend, backendHealthy, backendUnhealthy := config.DefaultKeepAlive()
+	backendUnhealthy.Timeout = 2 * time.Second
+	backendUnhealthy.Idle = time.Second
+	backendUnhealthy.Cnt = 1
+	backendUnhealthy.Intvl = time.Second
 	testTCPConn(t,
 		func(t *testing.T, cli *PacketIO) {
 			require.NoError(t, cli.SetKeepalive(frontend))
 			require.NoError(t, cli.ClientTLSHandshake(ctls))
+			// timeout by keepalive + timeout
+			time.Sleep(backendUnhealthy.Timeout)
+			_, err := cli.ReadPacket()
+			require.Error(t, err)
```

Comment:
I mean, cli sets timeout and srv sleeps 2s. And the read packet won't report error.